core/src/main/java/com/yugabyte/oss/driver/internal/core/loadbalancing/YugabyteDefaultLoadBalancingPolicy.java

```diff
@@ -216,10 +256,7 @@ private boolean handleNodeDownEvent(Node node) {
       handleSuccess = true;
     }
 
-    if (liveNodes.contains(node)) {
```

Comment:
As discussed, we can use return value of `remove()` to use for `handleSuccess`.

Reply:
Done.